update changelog (#2613)
Azure/azure-sdk: docs/policies/releases.md

{% include requirement/MUST %} name changelogs with all caps except for the extension, i.e. `CHANGELOG.md`.

> {% include requirement/MUST %} follow the [Change Logs](#change-logs) section below for how to format the file.

Reviewer: ```suggestion
{% include requirement/MUST %} follow the format in the guidelines section below [Change Logs](#change-logs).
```

{% include requirement/MUST %} name changelogs with all caps except for the extension, i.e. `CHANGELOG.md`.

{% include requirement/MUST %} follow the format below. Be sure to include the following headers that will be automatically added to your release notes:

- New Features - Adding a new API or feature
- Breaking Changes - Changes to an existing API or changes to the behavior of an existing API
- Key Bug Fixes - Major bug fixes that require a customer to do something, bug fixes that come from multiple up-votes in github or that are motivated by customer feedback

> - Security Fixes - Any security fix, no matter how small

Reviewer: @czubair  Should we remove "Security Fixes" as a header and just lump them under "Key Bug Fixes" I only as as I'd like to come up with a common template we can stamp out something like "

```
## <versionSpecifier> (Unreleased|YYYY-MM-DD)
### New Features

### Breaking Changes

### Key Bug Fixes
```

# Release History

## 12.1.0 (Unreleased)
### New Features
- check to enforce TokenCredential is used only over HTTPS

### Breaking Changes
- Support using SAS token from connection string

### Key Bug Fixes
- Issue where AccountName on BlobUriBuilder would not be populated
  for non-IP style Uris ([#8638](https://github.com/Azure/azure-sdk-for-net/issues/8638))

## 12.0.0 (2019-11-25)
### Breaking Changes
- Added support for the new low-priority node type.

> ### Renamed

Reviewer: I'm guessing Renamed is not something that we do often so perhaps we should remove this section but we should add other common headers like Added/Fixed/Changed etc like there was before.

Author: @weshaggard you want to add those on top of the sections that the automation will pull out?

Reviewer: For our automation it shouldn't matter where they are at as we will simply match the header name and pull the content under that header until we find another header at the same level or higher. 

However for the guidelines we want a nice example that folks can follow.

Author: Since folks already know the common labels they use, I think pointing specifically to what will be automated is the important point.

Reviewer: My goal is to try and create some sort of copy/paste template folks can use and fill out. So having a complete example and maybe even and empty example would be help for that.

# Release History

## 12.1.0 (Unreleased)
### Features Added
- check to enforce TokenCredential is used only over HTTPS

### Breaking Changes
- Support using SAS token from connection string

### Key Bugs Fixed
- Issue where AccountName on BlobUriBuilder would not be populated
  for non-IP style Uris ([#8638](https://github.com/Azure/azure-sdk-for-net/issues/8638))

## 12.0.0 (2019-11-25)
### Breaking Changes
- Added support for the new low-priority node type.
- Number of operations and models to better align with other client
  libraries and the .NET Framework Design Guidelines
  
 ### Fixed
- Parallel upload/download performance improvements
```

> {% include requirement/SHOULD %} link to GitHub issues (full URL) that were fixed in that release going forward (i.e. do not backfill older issues). See example above under the `### Key Bug Fixes` heading.

Reviewer: ```suggestion
{% include requirement/SHOULD %} link to GitHub issues (full URL) that were fixed in that release going forward (i.e. do not backfill older issues). See example above under the `### Key Bugs Fixed` heading.
```

We use GitHub releases as a convenient place to put release notes. The change log and any additional notes will be available here. We automatically publish release notes to a GitHub release if the changelog guidance below is followed. No artifacts are published to the GitHub release. Instead, use a supported package registry.

## Change Logs

Every library should contain a file exactly named `CHANGELOG.md` in the folder for each library. Generally in `sdk/<service>/<package>` folder. 

In order for consistency across our SDKs and to enable automation to validate and parse these changelogs the files must use the following format.

> 

Reviewer: ```suggestion
```

- Parallel upload/download performance improvements
```

> {% include requirement/SHOULD %} link to GitHub issues (full URL) that were fixed in that release going forward (i.e. do not backfill older issues). See example above under the `### KeyBugFixes` heading.

Reviewer: ```suggestion
{% include requirement/SHOULD %} link to GitHub issues (full URL) that were fixed in that release going forward (i.e. do not backfill older issues). See example above under the `### Key Bug Fixes` heading.
```
